pkgs/tools/misc/page/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "page";
+  version = "v2.3.2";
+
+  src = fetchFromGitHub {
+    owner = "I60R";
+    repo = pname;
+    rev = version;
+    sha256 = "1plnli0h9zcmncjn8xv1114lp4pw18hajcpmb1cbmhn43isqn3v5";
+  };
+
+  cargoSha256 = "0z1qlz0kpj2vpg3snl51qhmdgs71d2g6bs9mjkb11j545z9b0rrf";
+
+  meta = with stdenv.lib; {
+    description = "Use neovim as pager";
+    homepage = "https://github.com/I60R/page";
+    license = stenv.lib.licenses.mit;
+    platforms = platforms.all;
+    maintainer = stdenv.lib.maintainers.s1341;
```

Comment:
even if you are the only maintainer this should be plural and a list.

Reply:
done